### ISSUE TITLE
JEI Still display objects blocked by stage [More Tool] [Bug]

### ISSUE THREAD

**Issue by TheLynk** [Darkhax-Minecraft/ItemStages#70]
[img](https://ibb.co/9w1PJtL)
exemple

**Comment by Darkhax** [comment]
This is a duplicate of https://github.com/Darkhax-Minecraft/ItemStages/issues/61
